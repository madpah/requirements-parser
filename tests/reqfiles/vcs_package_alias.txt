SomeProject@git+https://git.repo/some_pkg.git@1.3.1
SomeProject@git+https://git.repo/some_pkg.git@1.3.1#egg=EggName
SomeProject@ git+https://git.repo/some_pkg.git@1.3.1
SomeProject @git+https://git.repo/some_pkg.git@1.3.1
SomeProject @ git+https://git.repo/some_pkg.git@1.3.1

-e SomeProject@git+https://git.repo/some_pkg.git@1.3.1

Some_Project@git+https://git.repo/some_pkg.git@1.3.1
Some-Project@git+https://git.repo/some_pkg.git@1.3.1
Some_Project@git+https://git.repo/some_pkg.git@1.3.1#egg=EggName
Some_Project@ git+https://git.repo/some_pkg.git@1.3.1
Some_Project @git+https://git.repo/some_pkg.git@1.3.1
Some_Project @ git+https://git.repo/some_pkg.git@1.3.1

-e Some_Project@git+https://git.repo/some_pkg.git@1.3.1

SomeProject[extra1,extra2]@git+https://git.repo/some_pkg.git@1.3.1
Some_Project[extra1]@git+https://git.repo/some_pkg.git@1.3.1
SomeProject[extra1,extra2]@git+https://git.repo/some_pkg.git@1.3.1#egg=EggName
SomeProject[extra1,extra2]@ git+https://git.repo/some_pkg.git@1.3.1
SomeProject[extra1,extra2] @git+https://git.repo/some_pkg.git@1.3.1
SomeProject[extra1,extra2] @ git+https://git.repo/some_pkg.git@1.3.1

-e SomeProject[extra1,extra2]@git+https://git.repo/some_pkg.git@1.3.1
