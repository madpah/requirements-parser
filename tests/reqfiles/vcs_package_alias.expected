[
 {
    "line": "SomeProject@git+https://git.repo/some_pkg.git@1.3.1",
    "editable": false,
    "local_file": false,
    "specifier": false,
    "vcs": "git",
    "revision": "1.3.1",
    "name": "SomeProject",
    "uri": "git+https://git.repo/some_pkg.git",
    "subdirectory": null,
    "path": null,
    "hash_name": null,
    "hash": null,
    "extras": [],
    "specs": []
 },
 {
   "line": "SomeProject@git+https://git.repo/some_pkg.git@1.3.1#egg=EggName",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "SomeProject@ git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "SomeProject @git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "SomeProject @ git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "-e SomeProject@git+https://git.repo/some_pkg.git@1.3.1",
   "editable": true,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },




 {
    "line": "Some_Project@git+https://git.repo/some_pkg.git@1.3.1",
    "editable": false,
    "local_file": false,
    "specifier": false,
    "vcs": "git",
    "revision": "1.3.1",
    "name": "Some_Project",
    "uri": "git+https://git.repo/some_pkg.git",
    "subdirectory": null,
    "path": null,
    "hash_name": null,
    "hash": null,
    "extras": [],
    "specs": []
 },
 {
    "line": "Some-Project@git+https://git.repo/some_pkg.git@1.3.1",
    "editable": false,
    "local_file": false,
    "specifier": false,
    "vcs": "git",
    "revision": "1.3.1",
    "name": "Some-Project",
    "uri": "git+https://git.repo/some_pkg.git",
    "subdirectory": null,
    "path": null,
    "hash_name": null,
    "hash": null,
    "extras": [],
    "specs": []
 },
 {
   "line": "Some_Project@git+https://git.repo/some_pkg.git@1.3.1#egg=EggName",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "Some_Project",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "Some_Project@ git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "Some_Project",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "Some_Project @git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "Some_Project",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "Some_Project @ git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "Some_Project",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },
 {
   "line": "-e Some_Project@git+https://git.repo/some_pkg.git@1.3.1",
   "editable": true,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "Some_Project",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": [],
   "specs": []
 },




 {
    "line": "SomeProject[extra1,extra2]@git+https://git.repo/some_pkg.git@1.3.1",
    "editable": false,
    "local_file": false,
    "specifier": false,
    "vcs": "git",
    "revision": "1.3.1",
    "name": "SomeProject",
    "uri": "git+https://git.repo/some_pkg.git",
    "subdirectory": null,
    "path": null,
    "hash_name": null,
    "hash": null,
    "extras": ["extra1", "extra2"],
    "specs": []
 },
 {
    "line": "Some_Project[extra1]@git+https://git.repo/some_pkg.git@1.3.1",
    "editable": false,
    "local_file": false,
    "specifier": false,
    "vcs": "git",
    "revision": "1.3.1",
    "name": "Some_Project",
    "uri": "git+https://git.repo/some_pkg.git",
    "subdirectory": null,
    "path": null,
    "hash_name": null,
    "hash": null,
    "extras": ["extra1"],
    "specs": []
 },
 {
   "line": "SomeProject[extra1,extra2]@git+https://git.repo/some_pkg.git@1.3.1#egg=EggName",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": ["extra1", "extra2"],
   "specs": []
 },
 {
   "line": "SomeProject[extra1,extra2]@ git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": ["extra1", "extra2"],
   "specs": []
 },
 {
   "line": "SomeProject[extra1,extra2] @git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": ["extra1", "extra2"],
   "specs": []
 },
 {
   "line": "SomeProject[extra1,extra2] @ git+https://git.repo/some_pkg.git@1.3.1",
   "editable": false,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": ["extra1", "extra2"],
   "specs": []
 },
 {
   "line": "-e SomeProject[extra1,extra2]@git+https://git.repo/some_pkg.git@1.3.1",
   "editable": true,
   "local_file": false,
   "specifier": false,
   "vcs": "git",
   "revision": "1.3.1",
   "name": "SomeProject",
   "uri": "git+https://git.repo/some_pkg.git",
   "subdirectory": null,
   "path": null,
   "hash_name": null,
   "hash": null,
   "extras": ["extra1", "extra2"],
   "specs": []
 }
]
